formver.compiler-plugin/core/src/org/jetbrains/kotlin/formver/core/embeddings/expression/ControlFlow.kt

```diff
@@ -222,6 +223,23 @@ data class MethodCall(val method: NamedFunctionSignature, val args: List<ExpEmbe
     override fun <R> accept(v: ExpVisitor<R>): R = v.visitMethodCall(this)
 }
 
+data class FunctionCall(val function: NamedFunctionSignature, val args: List<ExpEmbedding>) : DirectResultExpEmbedding {
+
+    override val type: TypeEmbedding = function.callableType.returnType
+
+    override val subexpressions: List<ExpEmbedding>
+        get() = args
+
+    override fun toViper(ctx: LinearizationContext): Exp = function.toFuncApp(
+        args.map { it.toViper(ctx) },
+        ctx.source.asPosition
+    )
+
+    override fun <R> accept(v: ExpVisitor<R>): R =
+        v.visitFunctionCall(this) // or however your visitor is set up
+
```

Comment:
nit: extra newline

Reply:
Unfortunately no, and it's not always an error - Kotlin doesn't have a strict policy, but it's good to be consistent and to keep the code reasonably compact (as it tends to be quite lengthy already)